ci/code_checks.sh

```diff
@@ -3182,14 +3182,6 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
         pandas.util.hash_pandas_object # There should be no backslash in the final line, please keep this comment in the last ignored function
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
-    MSG='Partially validate docstrings (SA05)' ;  echo $MSG
-    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=SA05 --ignore_functions \
```

Comment:
SA05 needs to be added to the list of codes above that validate all docstrings

Reply:
To clarify, I'll just need to change line 3186 to 
`$BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=SA05`